data-prepper-plugins/avro-codecs/build.gradle

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+plugins {
+    id 'java'
+}
+
+repositories {
+    mavenCentral()
+    maven { url 'https://jitpack.io'}
+}
+
+dependencies {
+    implementation project(path: ':data-prepper-api')
+    implementation group: 'org.apache.avro', name: 'avro', version: '1.11.1'
```

Comment:
Please use the string notation: `org.apache.avro:avro:1.11.1`.

Reply:
Done